doc/manual/src/installation/uninstall.md

```diff
@@ -135,6 +135,14 @@ which you may remove.
 
    If you _do_ find a "Nix Store" volume, delete it by running `diskutil deleteVolume` with the store volume's `diskXsY` identifier.
 
+8. Remove the password to the encrypted Nix Store volume:
+
+   ```console
+   sudo security delete-generic-password  -a "Nix Store" -s "Nix Store" -l "disk3 encryption password" -D "Encrypted volume password"`
```

Comment:
The instruction above uses diskXsY because disk3 is not a given. 

For example, mine is disk1s6. 

I'll also double-register that I don't think this is describing a credential that our installer adds. I suspect this might be from the detsys installer (or maybe our experimental version of that if someone's been handing out links to it?)

Reply:
I had a devenv 0.6 installation on my device. When I wanted to uninstall it I followed the guide https://devenv.sh/getting-started/ and noticed, that I did not run the suggested experimental installer when installing devenv 1.0. When I was not sure about the state of my installation I decided to wipe it and followed this guide. When I run the experimental installer I got the message, that I still have a password in my key chain that should be removed.

Good hint though, that the disk number varies. I could refer to the disk name from the previous step.